tests/mainEndpoints.spec.ts

```
@@ -0,0 +1,79 @@
+  const response = await request.get(mainConfigUrl!);
+  expect(response.status()).toEqual(200);
+  const responseObj = await response.json();
+  expect(responseObj.id).toEqual("9043acf9-2cf3-48ac-9656-a5d7c4b7593d");
+  expect(responseObj.name).toEqual("Kontur Atlas");
+  expect(responseObj.description).toEqual("Kontur SAAS application");
+  expect(responseObj.features[0]).toBeDefined();
+});
+
+test(`Check ${staticConfigUrl} availability`, async ({ request }) => {
+  expect(staticConfigUrl).toBeDefined();
+  const response = await request.get(staticConfigUrl!);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)